frontend/integration-tests/tests/kubevirt/vm.yaml.scenario.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+import { browser, ExpectedConditions as until } from 'protractor';
+
+import { appHost, testName } from '../../protractor.conf';
+import { isLoaded } from '../../views/crud.view';
+import { removeLeakedResources } from './utils';
+import { detailViewAction, detailViewVmIcon, runningIcon } from '../../views/kubevirt/vm.actions.view';
+import Yaml from './models/yaml';
+
+
+describe('Test create vm from yaml', () => {
+  const leakedResources = new Set<string>();
+  const yaml = new Yaml();
+
+  beforeEach(async() => {
+    await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+    await isLoaded();
+    await yaml.openYamlPage();
+  });
+
+  afterAll(async() => {
+    removeLeakedResources(leakedResources);
+  });
+
+  describe('Test create VM from the default yaml file', () => {
+    it('Creates VM', async() => {
+      await yaml.createVMFromYaml();
+      await isLoaded();
+      await detailViewAction('Start');
+      await browser.wait(until.presenceOf(detailViewVmIcon(runningIcon)));
+    });
+  });
+
+  describe('Test create VM from the default yaml file again', () => {
+    it('Fails to create VM because VM exists', async() => {
+      await yaml.createVMFromYaml();
+      await yaml.errorOccurOnCreateVM();
+      await yaml.cancelCreateVM();
+
```

Comment:
can you please check the cancel worked here?

Reply:
It works here.

Reply:
I think @suomiy's idea was to check whether you get redirected back to the list of VMs, something like ```await browser.wait(until.textToBePresentInElement(vmView.resourceTitle, 'Virtual Machines'), PAGE_LOAD_TIMEOUT);```

Reply:
please can you still check this in the testcase bellow?

Reply:
testcase below doesn't need this because it doesn't create the VM successfully.

Reply:
ik, just wanted to add the test for the button. not really important though

Reply:
add a check for the cancel button now.